qiskit_ibm_runtime/options/twirling_options.py

```diff
@@ -30,29 +30,34 @@ class TwirlingOptions:
     """Twirling options.
 
     Args:
-        gates: Whether to apply 2-qubit gate twirling.
-            By default, gate twirling is enabled for resilience level >0.
+        enable_gates: Whether to apply 2-qubit gate twirling. Default: False.
 
-        measure: Whether to apply measurement twirling.
-            By default, measurement twirling is enabled for resilience level >0.
+        enable_measure: Whether to apply measurement twirling. Default: True.
```

Comment:
This should be more clear that it only applys to expectation value measurements, not any measurements in the pub circuit.